src/controllers/handlers/picks-handlers.ts

```diff
@@ -1,43 +1,43 @@
-import { fromDBGetPickByItemIdToPickUserAddressesWithCount, TPick } from "../../adapters/picks"
-import { getNumberParameter, getPaginationParams } from "../../logic/http"
-import { InvalidParameterError } from "../../logic/http/errors"
-import { PickStats } from "../../ports/picks"
-import { HandlerContextWithPath, HTTPResponse, StatusCode } from "../../types"
+import { fromDBGetPickByItemIdToPickUserAddressesWithCount, TPick } from '../../adapters/picks'
+import { getNumberParameter, getPaginationParams } from '../../logic/http'
+import { InvalidParameterError } from '../../logic/http/errors'
+import { PickStats } from '../../ports/picks'
+import { HandlerContextWithPath, HTTPResponse, StatusCode } from '../../types'
 
 export async function getPickStatsHandler(
-  context: Pick<
-    HandlerContextWithPath<"picks", "/v1/picks/:itemId/stats">,
-    "url" | "components" | "params" | "request" | "verification"
-  >
+  context: Pick<HandlerContextWithPath<'picks', '/v1/picks/:itemId/stats'>, 'url' | 'components' | 'params' | 'request' | 'verification'>
 ): Promise<HTTPResponse<PickStats>> {
   const {
     url,
     components: { picks },
     verification,
-    params,
+    params
   } = context
   const userAddress: string | undefined = verification?.auth.toLowerCase()
 
   try {
-    const power = getNumberParameter("power", url.searchParams)
+    const power = getNumberParameter('power', url.searchParams)
 
-    const pickStats = await picks.getPickStats(params.itemId, { userAddress, power: power ?? undefined })
+    const pickStats = await picks.getPickStats(params.itemId, {
+      userAddress,
+      power: power ?? undefined
```

Comment:
the method `getNumberParameter` returns `undefined` when it doesn't find a query param. 
```suggestion
      power
```